sys/include/net/gnrc/rpl.h

```diff
@@ -513,8 +513,8 @@ uint8_t gnrc_rpl_gen_instance_id(bool local);
  */
 static inline void gnrc_rpl_config_pio(gnrc_rpl_dodag_t *dodag, bool status)
 {
-    dodag->req_opts = (dodag->req_opts & ~GNRC_RPL_REQ_OPT_PREFIX_INFO) |
-                      (status << GNRC_RPL_REQ_OPT_PREFIX_INFO_SHIFT);
+    dodag->dio_opts = (dodag->dio_opts & ~GNRC_RPL_DIO_REQ_OPT_PREFIX_INFO) |
```

Comment:
Good catch! Will address soon.
Am 15.03.2016 18:19 schrieb "BytesGalore" <notifications@github.com>:

> In sys/include/net/gnrc/rpl.h
> <https://github.com/RIOT-OS/RIOT/pull/5081#discussion_r56203361>:
>
> > @@ -513,8 +513,8 @@ uint8_t gnrc_rpl_gen_instance_id(bool local);
> >   */
> >  static inline void gnrc_rpl_config_pio(gnrc_rpl_dodag_t *dodag, bool status)
> >  {
> > -    dodag->req_opts = (dodag->req_opts & ~GNRC_RPL_REQ_OPT_PREFIX_INFO) |
> > -                      (status << GNRC_RPL_REQ_OPT_PREFIX_INFO_SHIFT);
> > +    dodag->dio_opts = (dodag->dio_opts & ~GNRC_RPL_DIO_REQ_OPT_PREFIX_INFO) |
>
> its GNRC_RPL_REQ_DIO_OPT_PREFIX_INFO_SHIFT
> REQ <-> DIO
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/5081/files#r56203361>
>